committers.html

```diff
@@ -597,6 +597,18 @@ <h1 class="content-title">The committers</h1>
           <a href="https://github.com/soarez">github.com/soarez</a><br>
         </td>
       </tr>
+      <tr>
```

Comment:
Instead of creating a new row, you can put your picture in a second cell on the right of Igor.
Otherwise it looks like
<img width="921" alt="image" src="https://github.com/apache/kafka-site/assets/903615/7c535755-011d-4a3c-8f18-9cb504f28ec5">

You can use `./start-preview.sh` to visualize changes.